pkgs/applications/science/biology/trf/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchFromGitHub, }:
+
+stdenv.mkDerivation rec {
+  pname = "trf";
+  version = "4.09.1";
+
+  src = fetchFromGitHub {
+    owner = "Benson-Genomics-Lab";
+    repo = pname;
```

Comment:
```suggestion
    repo = "trf";
```

Reply:
There are fewer examples of 'repo = pname;' in the nixpkgs code, so it seems better to define it explicitly.
Is there any documentation recommending this?

Reply:
>  since repo and pname are not the same semantically

Exactly, I'll remember it.